### PR TITLE
[Merged by Bors] - chore(MeasureTheory/Prod): minor golf, add a docstring

### PR DESCRIPTION
**Pull request by urkud**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/MeasureTheory/Constructions/Prod/Basic.lean [hunk 1]
```diff
@@ -779,22 +779,29 @@ namespace MeasurePreserving
...
 
+/-- Let `f : α → β` be a measure preserving map.
+For a.e. all `a`, let `g a : γ → δ` be a measure preserving map.
+Also suppose that `g` is measurable as a function of two arguments.
+Then the map `fun (a, c) ↦ (f a, g a c)` is a measure preserving map
+for the product measures on `α × γ` and `β × δ`.
+
+Some authors call a map of the form `fun (a, c) ↦ (f a, g a c)` a *skew product* over `f`,
+thus the choice of a name.
+-/
```
Comment (grunweg) [inline Mathlib/MeasureTheory/Constructions/Prod/Basic.lean:790]: You're using `a` here, and the theorem statement uses `x`. I like using `a` (or using `x` and renaming \alpha to `X`)... can you rename the variable in the hypothesis accordingly, please?

**Review by grunweg — COMMENTED** [review]
Thanks, looks good. I have one comment (and am not a measure theorist, so cannot comment on whether the term *skew product* actually exists) - after that, I'd maintainer merge this.

**Comment by urkud** [comment]
> Thanks, looks good. I have one comment (and am not a measure theorist, so cannot comment on whether the term _skew product_ actually exists) - after that, I'd maintainer merge this.

I know that my PhD advisor calls them "skew products". However, he is Russian, and sometimes he uses translations of Russian terms instead of existing English terms.

File: Mathlib/MeasureTheory/Constructions/Prod/Basic.lean (hunk 1 above)
Comment (urkud) [reply Mathlib/MeasureTheory/Constructions/Prod/Basic.lean:790]: Done.

**Comment by grunweg** [comment]
Thanks!
maintainer merge

**Comment by riccardobrasca** [comment]
Thanks!

bors merge

**Pull request closed without merging**